src/Altinn.Apps/KubernetesWrapper/KubernetesWrapper.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="KubernetesClient" Version="1.5.18" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Stylecop should be added to this project aswell

Reply:
Fixed :)